### DISCUSSION TITLE
ECMA Version?

### DISCUSSION THREAD

**Discussion by danFWD** [appcelerator/titanium_mobile#13779] [Q&A] [answered]
Does anyone know or can anyone point me where to find out what the current ECMA version that Ti supports?

**Comment by danFWD** [comment]
Currently running Node 14.15.0 so I am assuming this would be the ECMA (2015?) that it supports. I think I’ll just test out some newer syntax as my project was originally built in 2015 and used VERY old JS syntax.

**Reply by m1ga** [reply]
you can use up to node 16 and the (unreleased) master branch supports node 19 already. ES6 was added back in 2018 https://blog.axway.com/learning-center/software-development/api-development/using-es6-in-a-titanium-app and is enabled by default for a while now (so you don't need the transpile part in the blog post)

**Comment by m1ga** [comment, marked as answer]
I know you can use ES6 to write your code but I can't give you a specific version number or file to look for. Maybe someone else knows how to find the exact version.
